src/test/java/com/hyperwallet/clientsdk/model/HyperwalletUserTest.java

```diff
@@ -12,6 +14,7 @@ protected HyperwalletUser createBaseModel() {
         user
                 .token("test-token")
                 .status(HyperwalletUser.Status.ACTIVATED)
+                .verificationStatus(VerificationStatus.VERIFIED)
                 .createdOn(new Date())
                 .clientUserId("test-client-user-id")
                 .profileType(HyperwalletUser.ProfileType.INDIVIDUAL)
```

Comment:
Please add verification on the ff: 
1) validating `.verificationStatus(VerificationStatus.VERIFIED)` input from mocked request body 
2) validating  test for this code ` public HyperwalletUser clearVerificationStatus() {
        clearField("verificationStatus");
        verificationStatus = null;
        return this;
    }`

if it already exist please disregard this comment thanks